src/main/java/uk/gov/pay/products/resources/ProductResource.java

```diff
@@ -54,7 +54,20 @@ public Response createProduct(JsonNode payload) {
     @Consumes(APPLICATION_JSON)
     public Response deprecatedFindProduct(@PathParam("productExternalId") String productExternalId) {
         logger.info("Find a product with externalId - [ {} ]", productExternalId);
```

Comment:
If you rebase you will find that this method is not deprecated anymore hence we probably want to update it again...